<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>320</width>
     <height>680</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 290 12pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>55</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>320</width>
      <height>540</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>320</width>
        <height>530</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget::tab {
    background-color: rgb(255, 255, 255);
    border-style: solid;
    border-color: rgb(255, 255, 255);
    border-bottom-width: 1px;
    border-bottom-color: rgb(255, 255, 255);
    height: 30px;
    width: 160px;
}

QWidget::tab:selected {
    border-bottom-color: rgb(18, 183, 245);
    border-bottom-width: 2px;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QListWidget" name="listWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>320</width>
          <height>530</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget::Item{
background-color: rgb(255, 255, 255);
}

QListWidget::Item:hover{
background-color:rgb(242, 242, 242);
}</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>320</width>
          <height>510</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget::tab{
height:30px;
width:60px;
border:none;
}
</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QListWidget" name="listWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>320</width>
            <height>480</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QListWidget" name="listWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>320</width>
            <height>480</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(99, 188, 255);
color: rgb(255, 255, 255);
font-size:14px;
}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>10</y>
        <width>320</width>
        <height>90</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>232</x>
        <y>38</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(99, 188, 255);
color: rgb(255, 255, 255);
font-size:14px;
}</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 198, 253);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>29</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(75, 217, 254);
border-image: url(:/images/s.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/lib/search.png</normaloff>:/lib/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>272</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(75, 217, 254);
color: rgb(0, 0, 0);
border: none; 




</string>
       </property>
       <property name="placeholderText">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/suoxiao.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/yx_x.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>toolButton_2</zorder>
   <zorder>toolButton_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
